src/hffs/fs.py

```diff
@@ -182,40 +206,41 @@ def _open(
         **kwargs,
     ):
         if mode == "ab":
-            raise NotImplementedError("Appending to remote files is not supported")
+            raise NotImplementedError("Appending to remote files is not yet supported.")
         path = self._strip_protocol(path)
         return HfFile(self, path, mode=mode, **kwargs)
 
-    def _rm(self, path):
+    def _rm(self, path, **kwargs):
         path = self._strip_protocol(path)
         operations = [huggingface_hub.CommitOperationDelete(path_in_repo=path)]
-        commit_message = f"Delete {path} with hffs"
+        commit_message = f"Delete {path}"
         self._api.create_commit(
             repo_id=self.repo_id,
             repo_type=self.repo_type,
             token=self.token,
             operations=operations,
             revision=self.revision,
-            commit_message=commit_message,
+            commit_message=kwargs.get("commit_message", commit_message),
+            commit_description=kwargs.get("commit_description", None),
```

Comment:
```suggestion
            commit_description=kwargs.get("commit_description"),
```
`None` could be implicit here